env/netbox.env

```diff
@@ -1,4 +1,5 @@
 CORS_ORIGIN_ALLOW_ALL=True
+CSRF_TRUSTED_ORIGINS=http://netbox.example.com/*
```

Comment:
I believe that a sensible default would just be an empty value here.

Reply:
Removed from netbox.env